pytensor/sparse/variable.py

```diff
@@ -191,9 +200,8 @@ def __floor__(self):
     def __trunc__(self):
         raise NotImplementedError
 
-    @override_dense
     def transpose(self):
-        raise NotImplementedError
+        return self.T
```

Comment:
Do you want to add the `mT` property as well? Will make code that works with tensors more likely to work with sparse variables as well